### PR TITLE
Holmes-Toolbox Universal Installer

### LINKED ISSUE #2: Add start up scripts
We need some start up scripts for Totem and Totem Services

### LINKED ISSUE #7: Add libmagic to storer install script
SSIA

### PR DESCRIPTION
**Pull request by ms-xy**
As the systems complexity grows and more people get involved an easy way to set up a working system is required.
The universal-installer.sh can install RabbitMQ, Cassandra, Holmes-Storage and Holmes-Totem on Ubuntu/Debian systems.

- supports installing Totem/Storage as services for Upstart or Systemd
- Holmes-Storage configuration files can be generated

resolves #7
resolves #2

### CODE REVIEW

File: linux-installer/storage/config_helper.go [hunk 1]
```diff
@@ -0,0 +1,402 @@
...
+            
+        default:
+            fmt.Println("Invalid option for --config= supplied.")
+            panic("Invalid option for --config= supplied.")
+    }
+    
+    // gather various other information
+    fmt.Println("> Other settings:")
+    configObj.LogFile       = get_text(msg_logfile, "")
+    configObj.LogLevel      = get_text_generic(msg_loglevel, "debug", "error/warn/info/debug")
```
Comment (cynexit) [inline linux-installer/storage/config_helper.go]: only supported log levels are "warning/info/debug", anything else will default to "debug"

**Comment by webstergd** [comment]
This looks pretty good. Can you please update the readme.md file to discuss how to work with this?

**Comment by cynexit** [comment]
Looks good, to fully resolve #2 please add a new folder "startup-scripts" to the root of this repository and also copy your templates into there, many people will look for them separately.
Also, please add a section to the main README.md so people know what this is and how to use it. :)

**Comment by webstergd** [comment]
I agree that we should do something to make the startup-scripts easier. We also need a manual guide

option 1:
Push the start up scripts to the core repos. The install script should then pull from this location and set everything up. We should also update the readme.md files for each repo on how to access and install them. The core documentation should also list this. 

option 2:
Leave these as is. In the core documentation explain how to perform a manual install. Have a section that locates the start up scripts and shows how to manually install them without the guide.

Thoughts?

**Comment by ms-xy** [comment]
I would favor option 2. If we now split part of it up into the core repos again, that would defy the point of gathering it all in one repository for maintainability and central point of access.

@cynexit users will not look into the toolbox searching for startup scripts for Totem/Storage without mentioning it in the documentation anyways. As such I would suggest we keep the templates where they are right now and make a section in the documentation about finding and installing them.

File: linux-installer/storage/config_helper.go (hunk 1 above)
Comment (ms-xy) [inline linux-installer/storage/config_helper.go]: thx, I'll fix that

**Comment by ms-xy** [comment]
Fixed the log prio and added README.md change / start-script change as requested.

**Comment by webstergd** [comment]
Danke!

**Pull request merged**